backend/core/views/auth.py

```diff
@@ -163,20 +166,11 @@ def do_logout(request):
 
 @require_GET
 def get_total_score(request, uuid):
-    player = get_object_or_404(Player, pk=uuid)
-    total_score = Vote.objects.filter(pad_step__player=player).count()
-    full_ranking = (
-        Player.objects.annotate(vote_count=Count("steps__votes"))
-        .filter(vote_count__gt=0)
-        .order_by("-vote_count")
-    )
-
-    try:
-        my_ranking = [
-            index + 1
-            for index in range(len(full_ranking))
-            if full_ranking[index].name == player.name
-        ][0]
-    except IndexError:  # Player is not in ranking
-        my_ranking = None
-    return JsonResponse({"score": total_score, "ranking": my_ranking})
+    players = Player.objects.annotate(
+        rank=Window(expression=Rank(), order_by=F("total_score").desc())
+    ).all()
+
+    for player in players:
+        if str(player.uuid).replace("-", "") == uuid:
```

Comment:
player.uuid.hex = ce que tu cherches
```suggestion
        if player.uuid.hex == uuid:
```